morph/cli.py

```diff
@@ -171,6 +175,30 @@ def cmd_grep(args):
                         return
 
 
+def cmd_sync_known(args):
+    filenames = args.input or ['known.txt']
+    should_merge = args.merge
+
+    external_db = load_db('external', allow_missing=True)
+    if not should_merge:
+        for m, locs in external_db.db.iteritems():
+            locs.difference_update([loc for loc in locs
+                                    if isinstance(loc, morph.morphemes.Nowhere)])
+
+    db_dir = os.path.join(profile_path(), 'dbs')
+    loc = morph.morphemes.Nowhere(maturity=30)  # arbitrary but > maturity threshold
```

Comment:
Why does it have to be greater than the maturity threshold?
Other question: isn't Nowhere and Corpus redundant/isn't it better drop Nowhere (isn't used anywhere as far as I can tell) in favor of Corpus?